lib/Onyx.js

```diff
@@ -141,7 +141,7 @@ function getAllKeys() {
  * @returns {Boolean}
  */
 function isCollectionKey(key) {
-    return _.contains(_.values(onyxKeys.COLLECTION), key);
+    return onyxCollectionKeyMap.has(key);
```

Comment:
This is causing the jest tests to fail: https://github.com/Expensify/App/actions/runs/6027173662/job/16353221477